### PR TITLE
PHPdoc for the `Shortcode_UI` class

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #389

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@goldenapples #reviewmerge

File: inc/class-shortcode-ui.php [hunk 1]
```diff
@@ -1,37 +1,80 @@
...
+	 * Shortcake controller instance.
+	 *
+	 * @access private
+	 * @var object
+	 */
 	private static $instance;
 
+	/**
+	 * Get instance of Shortcake controller.
+	 * Instantiates object on the fly when not already loaded.
```
Comment (goldenapples) [inline inc/class-shortcode-ui.php:43]: Blank  line between summary & description?

Comment (danielbachhuber) [inline inc/class-shortcode-ui.php:43]: Good catch. Updated in cc33354

File: inc/class-shortcode-ui.php [hunk 2]
```diff
@@ -124,37 +188,48 @@ public function enqueue() {
 			),
 		) );
 
-		// queue templates
+		// add templates to the footer, instead of where we're at now
 		add_action( 'admin_print_footer_scripts', array( $this, 'action_admin_print_footer_scripts' ) );
 
+		/**
+		 * Fires after shortcode UI assets have been enqueued.
+		 * Will only fire once per page load.
```
Comment (goldenapples) [inline inc/class-shortcode-ui.php:197]: Blank line here as well?

**Comment by goldenapples** [comment]
I'm not sure from reading the docs whether the blank lines are necessary between summary & description in hook documentation, but it seems best to be consistent. If you fix the one remaining spot missing the blank line, I'll merge.

**Pull request merged**